containers/images/pulp/container-assets/route_paths.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import pkgutil
+import json
+import re
+import sys
+import nginx
+from pathlib import Path
+
+nginx_configs = []
+
+for i in pkgutil.iter_modules():
+    # filter pulp and galaxy packages
+    if re.search(r'pulp_.*|galaxy_ng',i.name):
+        # only add the filename to array if it exists
+        if Path(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf").is_file():
+            nginx_configs.append(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf")
+
+name = sys.argv[1]
+router = []
+
+for plugin_conf in nginx_configs:
+    app = "container"
```

Comment:
we should get a descriptive name of the plugin here,
For example: 
pulp-ansible -> ansible
pulp-container -> container

Reply:
```suggestion
    app = "galaxy" if "galaxy" in plugin_conf else plugin_conf.split("pulp_")[1].split("/")[0]
```